components/ShareWidget.vue

```diff
@@ -0,0 +1,174 @@
+    ]"
+  >
+    <div class="row">
+      <div class="column">
+        <a
+          href="https://twitter.com/share?ref_src=twsrc%5Etfw&text=Up%20to%20the%20minute%20Bay%20Area%20COVID%20statistics%20and%20county%20information:&url=https://panda.baybrigades.org/"
+          data-show-count="false"
+        >
+          <div class="twitter widge">
+            <i class="fab fa-twitter fa-2x" />
+          </div>
+        </a>
+        <script
+          async
+          src="https://platform.twitter.com/widgets.js"
+          charset="utf-8"
+        />
+      </div>
+
+      <div class="column">
+        <a
+          href="https://www.facebook.com/sharer/sharer.php?u=https://panda.baybrigades.org/"
+          target="_blank"
+        >
+          <div class="facebook widge">
+            <i class="fab fa-facebook-f fa-2x" />
+          </div>
+        </a>
+      </div>
+
+      <div class="column">
+        <a
+          href="http://www.reddit.com/submit?url=https://panda.baybrigades.org/"
+        >
+          <div class="reddit widge">
+            <i class="fab fa-reddit-alien fa-2x" />
+          </div>
+        </a>
+      </div>
+
+      <div class="column">
+        <a
+          href="http://www.linkedin.com/shareArticle?mini=true&url=https://panda.baybrigades.org/"
+        >
+          <div class="linkedin widge">
+            <i class="fab fa-linkedin-in fa-2x" />
+          </div>
+        </a>
+      </div>
+
+      <div v-if="navigatorShare" class="column">
+        <div id="emailWidget" class="email widge" @click="share">
+          <i class="fas fa-share-alt fa-2x" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ShareWidget',
+  data() {
+    return { navigatorShare: true, onMain: true }
+  },
+  mounted() {
+    this.onMain =
+      location.pathname === '/main' || location.pathname === '/about'
+    this.navigatorShare = navigator.share !== undefined
+  },
+  methods: {
+    share() {
+      if (navigator.share) {
+        navigator.share({
+          title: 'Bay Area Pandemic Dashboard',
+          url: 'https://panda.baybrigades.org/'
+        })
+      }
+    }
+  }
+}
+</script>
+
+<style lang="css" scoped>
+.ShareWidget {
+  box-sizing: inherit;
+  width: 100%;
+  position: fixed;
+  bottom: 0px;
+  z-index: 100;
+}
+.row {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+}
+.column {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+a {
+  text-decoration: none;
+  font-size: 0.9rem;
+}
+a:hover {
+  text-decoration: none;
+}
+.widge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 45px;
+  color: white;
+}
+.twitter {
+  background-color: #56acee;
+}
+.facebook {
+  background-color: #3b5998;
+}
+.reddit {
+  background-color: #ff3f19;
+}
+.linkedin {
+  background-color: #007ab5;
+}
+.email {
+  background-color: #473a8c;
+}
+.email:hover {
+  cursor: pointer;
+}
+
+@media (min-width: 1440px) {
```

Comment:
Can you set this to 600px? That way the share widgets only show up on small devices.

Reply:
I can, but just so I understand correctly... That would mean there would be no share options on Desktop? Currently there are supposed to be options for desktop as well per the issue spec. I can of course remove them and that would mean I could get rid of a lot of the css that is specifically for positioning on desktop, just would like to confirm. See pic below or in issue #519 thread

<img width="1408" alt="DesktopStyle" src="https://user-images.githubusercontent.com/39514323/105751251-9b94fe80-5efa-11eb-8b03-ef370db7e5f5.png">

Reply:
The share options showing on desktop screen larger than 1440px is perfect as you've shown in the screenshot above. It should continue to show that way all the way down to 601px.

We just want the share options showing at the bottom to show up when the screen is smaller than 600px.
![Screen Shot 2021-01-25 at 10 55 32 AM](https://user-images.githubusercontent.com/5452152/105752286-f1b67180-5efb-11eb-9f2c-08076efe1005.png)

Reply:
Ah sorry for my confusion, I see what you mean now. I was just setting the breakpoint by eye, I will update this as specified.

Reply:
No need to be sorry, we're all learning. :)

Since you're using css instead of scss, `<style lang="css" scoped>`, you don't get to take advantage of the mixins such as `@include lessThan($small)`. The small is a break point variable defined in `/assets/variables.scss`. You can check out the other break points there for reference.

Reply:
Oh, thanks! I was wondering where $small was defined. I'll take a look around but I'm not very good with scss so I should probably leave it as is if that's okay.

Reply:
yea css is totally fine.